.slashrc

```diff
@@ -326,15 +326,22 @@ config = os.environ.get(
   "VAAPI_FITS_CONFIG_FILE",
   os.path.abspath(os.path.join(media.mypath, "config", "default")))
 assert os.path.exists(config)
-execfile(config)
+
+global_namespace = {
+  "__file__": config,
+  "__name__": "__main__",
+}
+local_namespace = locals()
+with open(config, 'rb') as file:
+  exec(file.read(), global_namespace, local_namespace)
```

Comment:
Can we use default globals and locals parameters... that is, `exec(file.read())` should be enough?

Reply:
did the change